### ISSUE TITLE
Adjust theme for high contrast mode

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#3186]
### Describe the Problem
Adjust theme to use high contrast mode colors in build-configs

### Describe Your Preferred Solution

ThemeType and ColorType should be adjusted in build-configs to be available for further usage and adjustments.

**Comment by steffenkleinle** [comment]
Done in #3194
